test-base.cfg

```diff
@@ -215,7 +221,7 @@ initialization =
         shell=True, stdout=subprocess.PIPE).stdout.read().strip()
     if '--absolute' in sys.argv: pkgdir = os.path.abspath(pkgdir) ; sys.argv.remove('--absolute')
     if len(sys.argv) < 2: sys.argv.extend([pkgdir])
-    sys.argv.extend(${pycodestyle:args})
+    sys.argv.extend(${pycodestyle-ci-run:args})
```

Comment:
The best I would offer for your workflow is reverse-i-search `^R pycode`.

The core issue at hand here is that editors and IDEs blindly assume the executable name.